Documentation/developer-setup.md

```diff
@@ -23,7 +23,7 @@ There are detailed instructions for setting up a Cesium for Unreal development e
 
 * [Windows](developer-setup-windows.md)
 * [Linux](developer-setup-linux.md)
-* macOS - coming soon
+* [Mac OS X](developer-setup-osx.md)
```

Comment:
Pretty sure Apple doesn't call it this anymore. It's just `macOS` now.

Reply:
@kring Good catch. Fixed throughout.